gooddata-sdk/gooddata_sdk/catalog/workspace/declarative_model/workspace/workspace.py

```diff
@@ -54,30 +58,47 @@ def __init__(
         compute_client: str = None,
         model: CatalogDeclarativeWorkspaceModel = None,
         parent: CatalogWorkspaceIdentifier = None,
+        permissions: list[CatalogDeclarativeSingleWorkspacePermission] = None,
+        hierarchy_permissions: list[CatalogDeclarativeWorkspaceHierarchyPermission] = None,
     ):
         super(CatalogDeclarativeWorkspace, self).__init__(id, name)
         self.compute_client = compute_client
         self.model = model
         self.parent = parent
+        self.permissions = permissions if permissions is not None else []
+        self.hierarchy_permissions = hierarchy_permissions if hierarchy_permissions is not None else []
 
     @classmethod
     def from_api(cls, entity: dict[str, Any]) -> CatalogDeclarativeWorkspace:
         return cls(
-            entity["id"],
-            entity["name"],
-            entity.get("computeClient"),
-            CatalogDeclarativeWorkspaceModel.from_api(entity["model"]) if "model" in entity else None,
-            CatalogWorkspaceIdentifier.from_api(entity["parent"]) if "parent" in entity else None,
+            id=entity["id"],
+            name=entity["name"],
+            compute_client=entity.get("computeClient"),
+            model=CatalogDeclarativeWorkspaceModel.from_api(entity["model"]) if "model" in entity else None,
+            parent=CatalogWorkspaceIdentifier.from_api(entity["parent"]) if "parent" in entity else None,
+            permissions=[
```

Comment:
Could not you use here CatalogDeclarativeWorkspacePermission**s**?

Reply:
Irrelevant. Solved off-line with Jacek :)